src/fitnesseMain/badCodeToTestCheckstyle.java

```diff
@@ -0,0 +1,53 @@
+package fitnesseMain;
+
+import java.util.*; //BAD: star imports
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Using the '.*' form of import should be avoided - java.util.*. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck)